Hailstone.java

```diff
@@ -10,7 +11,24 @@
 
 public class Hailstone extends ConsoleProgram {
 	public void run() {
-		/* You fill this in */
+		println("This program finds the largest and smallest numbers");
```

Comment:
I assume you copied this from the Find Range problem because it is not true that this program finds the largest and smallest numbers.